enterprise/coderd/license/license.go

```diff
@@ -176,7 +176,7 @@ type Features struct {
 	BrowserOnly    int64 `json:"browser_only"`
 	SCIM           int64 `json:"scim"`
 	WorkspaceQuota int64 `json:"workspace_quota"`
-	RBAC           int64 `json:"rbac"`
+	TemplateRBAC   int64 `json:"template_rbac"`
```

Comment:
This will need to be changed in `coder/license` too

Reply:
Which files do you mean exactly?

Reply:
Well, right now it's authorized as `rbac` in that repo, so a license couldn't be generated that would make this feature enabled.

Reply:
https://github.com/coder/license/pull/9

Reply:
wait this is for both template ACLs and groups? shouldn't we split it out?

Reply:
groups are pretty useless with template acl and vice versa